cmd/containerd/config_linux.go

```diff
@@ -2,16 +2,15 @@ package main
 
 func defaultConfig() *config {
 	return &config{
-		Root:  "/var/lib/containerd",
-		State: "/run/containerd",
+		Root: "/var/lib/containerd",
 		GRPC: grpcConfig{
 			Address: "/run/containerd/containerd.sock",
 		},
 		Debug: debug{
 			Level:   "info",
 			Address: "/run/containerd/debug.sock",
 		},
-		Snapshotter: "overlay",
-		Differ:      "base",
+		Snapshotter: "io.containerd.snapshotter.v1.overlayfs",
+		Differ:      "io.containerd.differ.v1.base-diff",
```

Comment:
Will try and come up with a better name and change after this PR, are you suggesting `smart` 😛